modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,21 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+
+=== Spell Checker Pro 3.3.0
+
+The {productname} 6.5 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+Spell Checker Pro 3.3.0 adds one change:
+
+=== Words containing non-Latin characters were not sent in spelling requests.
+//#TINY-9654
+
+As of Spell Checker Pro 3.2.1, only languages using a Latin-based alphabet, including English, French, German, etc., had supported dictionaries. As a result, Spell Checker Pro's client side only sent words it “understands” in spelling requests to the server, omitting words containing non-Latin characters from languages like Greek, Arabic, and Korean. Specifically, the word identification logic used the regular expression `/[\w'\\-\u00AD\u0100-\u017F\u00C0-\u00FF\uFEFF\u2018\u2019]/` to capture characters including the extended Latin alphabet. This interfered with the usages of some customers pairing the Spell Checker Pro client side with a custom spellchecking service with support for languages with non-Latin alphabets, as words containing non-Latin characters would not be sent to the server.
```

Comment:
```suggestion
As of Spell Checker Pro 3.2.1, only languages using a Latin-based alphabet, including English, French, German, etc., had supported dictionaries.

As a result, Spell Checker Pro's client side only sent words it “understands” in spelling requests to the server, omitting words containing non-Latin characters from languages like Greek, Arabic, and Korean.

This interfered with the usages of some customers pairing the Spell Checker Pro client side with a custom spellchecking service with support for languages with non-Latin alphabets, as words containing non-Latin characters would not be sent to the server.
```
Split up the paragraph, and I'm not sure how relevant mentioning the regular expressions is?
BTW, just checking this ticket since its the only other Spellchecker ticket besides TINY-9632, so we should modify the heading on line 38 from `Spell Checker Pro 3.3.0 adds one change:` to `Spell Checker Pro 3.3.0 includes one improvement:` cos there's already an `adds` heading.

Reply:
@MitchCTiny Thanks for the above, regarding the headings... when merging any incoming tickets we will be updating any existing headings that have already been added to the release notes from previous tickets. _As a precaution, we also have a closing `copy edit` ticket that will be for final edits to `staging` before we merge back to `release/docs-6`_

@mkzhx If you are happy with the above change, please advise so i can add the commit suggestion.